### PR TITLE
Improve install command by asking for installing optional components.

### PR DESCRIPTION
**Pull request by pepov**
Also do not enable auditsink and cert-manager by default.

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | yes (auditsink/certmanager disabled by default)
| Deprecations?   | no 
| License         | Apache 2.0

### What's in this PR?
Improves the install command by asking questions to optionally install everything even if `-a` was not provided.

### Why?
To allow the user to decide what components to install without knowing about the `-a` flag. This will help us simplify the docs as well.

### CODE REVIEW

File: internal/cli/cmd/install.go [hunk 1]
```diff
@@ -403,66 +406,120 @@ func (c *installCommand) certManagerRunning() (exists bool, healthy bool, err er
...
+	c.shouldInstallCanary = installCanaryExplicitly || installCanaryInteractively
+
+	installDemoExplicitly := options.installDemoapp || options.installEverything
+	installDemoInteractively := false
+
+	if (!options.installDemoapp || !installDemoExplicitly) && c.cli.InteractiveTerminal() {
+		err := survey.AskOne(&survey.Confirm{
+			Renderer: survey.Renderer{},
+			Message:  "Install demo application (optional). Press enter to accept",
+			Default:  true,
```
Comment (waynz0r) [inline internal/cli/cmd/install.go]: this should be false by default

File: internal/cli/cmd/install.go [hunk 2]
```diff
@@ -403,66 +406,120 @@ func (c *installCommand) certManagerRunning() (exists bool, healthy bool, err er
...
+	c.shouldRunDemo = installDemoExplicitly || installDemoInteractively
+
+	runDemoExplicitly := options.runDemo || options.installEverything
+	runDemoInteractively := false
+
+	if (!options.runDemo || !runDemoExplicitly) && c.cli.InteractiveTerminal() {
+		err := survey.AskOne(&survey.Confirm{
+			Renderer: survey.Renderer{},
+			Message:  "Run demo application (optional). Press enter to accept",
+			Default:  true,
```
Comment (waynz0r) [inline internal/cli/cmd/install.go]: this should be false by default

File: internal/cli/cmd/install.go [hunk 3]
```diff
@@ -403,66 +406,120 @@ func (c *installCommand) certManagerRunning() (exists bool, healthy bool, err er
...
+		err := survey.AskOne(&survey.Confirm{
+			Renderer: survey.Renderer{},
+			Message:  "Install demo application (optional). Press enter to accept",
+			Default:  true,
+		}, &installDemoInteractively)
+		if err != nil {
+			return err
+		}
+	}
+	c.shouldRunDemo = installDemoExplicitly || installDemoInteractively
```
Comment (waynz0r) [inline internal/cli/cmd/install.go]:
```suggestion
	c.shouldInstallDemo = installDemoExplicitly || installDemoInteractively
```

File: internal/cli/cmd/install.go [hunk 4]
```diff
@@ -403,66 +406,120 @@ func (c *installCommand) certManagerRunning() (exists bool, healthy bool, err er
...
-	dbOptions := NewDashboardOptions()
-	dbOptions.QueryParams["namespaces"] = demoapp.GetNamespace()
-	dbCmd := NewDashboardCommand(cli, dbOptions)
-	err = dbCmd.RunE(dbCmd, nil)
-	if err != nil {
-		return errors.WrapIf(err, "error during opening dashboard")
+	installCanaryExplicitly := options.installCanary || options.installEverything
+	installCanaryInteractively := false
+
+	// audit-sink requires cert-manager as a dependency
```
[the remark this answers is not in the document]
Comment (pepov) [reply internal/cli/cmd/install.go]: yes

**Pull request merged**